ovs/dal/hybrids/vdisk.py

```diff
@@ -296,3 +300,15 @@ def calculate_delta(key, dynamic, current_stats):
                 else:
                     current_stats['{0}_ps'.format(key)] = max(0, (current_stats[key] - previous_stats[key]) / delta)
         volatile.set(prev_key, current_stats, dynamic.timeout * 10)
+
+    def reload_clients(self, client):
+        """
+        Reloads the StorageDriverClient and ObjectRegistryClient
+        """
+        if self.vpool_guid:
+            self._frozen = False
```

Comment:
The ```self._frozen``` is used internal to prevent (some) properties from being set. Not (temporarily) setting it to ```False``` would cause an error when the clients are changed. It's a I-know-what-I'm-doing-here flag.